### PR TITLE
[Lite] num_ranks option brings to user through cmd line parameter

### PR DESCRIPTION
**Pull request by Dayananda-V**
num_ranks optional parameters provides to user to calculate accuracy ranks on out file.

### CODE REVIEW

File: tensorflow/lite/tools/accuracy/ilsvrc/README.md [hunk 1]
```diff
@@ -56,6 +56,10 @@ and the following optional parameters:
     Optionally, the computed accuracies can be output to a file as a
     string-serialized instance of tflite::evaluation::TopkAccuracyEvalMetrics.
 
+*   `num_ranks`: `int` (default=10) \
+    The number of top accuracy counts, if 5, top-1 to top-5 rank result will
```
Comment (srjoglekar246) [inline tensorflow/lite/tools/accuracy/ilsvrc/README.md]: Please format as: "The number of top-K accuracies to return. For example, if num_ranks=5, top-1 to top-10 accuracy fractions are returned."

File: tensorflow/lite/tools/accuracy/ilsvrc/imagenet_model_evaluator.cc [hunk 2]
```diff
@@ -144,6 +145,9 @@ class CompositeObserver : public ImagenetModelEvaluator::Observer {
       tflite::Flag::CreateFlag(kDelegateFlag, &params.delegate,
                                "Delegate to use for inference, if available. "
                                "Must be one of {'nnapi', 'gpu'}"),
+      tflite::Flag::CreateFlag(kNumRanksFlag, &params.num_ranks,
+                               "Generates the top-1 to top-k accuracy counts "
+                               "where k is number of rank. Default: 10"),
```
Comment (srjoglekar246) [inline tensorflow/lite/tools/accuracy/ilsvrc/imagenet_model_evaluator.cc]: `top-k accuracy values` since it returns fractions & `where k = num_ranks`.

**Review by srjoglekar246 — CHANGES_REQUESTED** [review]
Just a couple of minor nits, should be ready to go then. Thanks for the fixes!

File: tensorflow/lite/tools/accuracy/ilsvrc/README.md (hunk 1 above)
Comment (Dayananda-V) [reply tensorflow/lite/tools/accuracy/ilsvrc/README.md]: fixed.

File: tensorflow/lite/tools/accuracy/ilsvrc/imagenet_model_evaluator.cc (hunk 2 above)
Comment (Dayananda-V) [reply tensorflow/lite/tools/accuracy/ilsvrc/imagenet_model_evaluator.cc]: fixed.

**Review by srjoglekar246 — APPROVED** [review]

**Pull request merged**